components/Internship/CommitteeInternshipInfo.js

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+import React, {useState} from 'react';
+
+import styles from '../../styles/components/Internship/CommitteeInternshipInfo.module.scss';
+
+function CommitteeInternshipInfo(props) {
+    const {committees} = props;
+    const [committeeName, setCommitteeName] = useState('Board');
+    return(
+        <>
+            <div className={styles['logos-line']}>
+                {committees.map(committee =>
+                    <div className={styles['logo-diamond']} key={committee.name} onClick={()=>setCommitteeName(committee.name)}>
+                        <Image
+                            src={committee.diamond}
+                            alt={`${committee.name}'s logo`}
```

Comment:
In this case, this is not good alt text, since we really just want this to read as "ACM Hack", etc.